src/fsharp/SyntaxTree.fs

```diff
@@ -695,6 +695,7 @@ type SynExpr =
     /// F# syntax: do expr
     | Do of
         expr: SynExpr *
```

Comment:
this makes me wish that we could include parameter-level documentation on DU members specifically and have them rolled-up somehow.